cpp/src/bitmask/null_mask.cu

```diff
@@ -222,6 +271,40 @@ cudf::size_type count_unset_bits(bitmask_type const *bitmask, size_type start,
   return (num_bits - detail::count_set_bits(bitmask, start, stop, stream));
 }
 
+// Create a bitmask from a vector of column views
+void concatenate_masks(std::vector<column_view> const &views,
+    bitmask_type * dest_mask,
```

Comment:
Why is this using an output parameter instead of returning a `device_buffer`?

Reply:
String view concatenate preallocates the null masks. This function exists so that it can overwrite that preallocated buffer. Moreover, this is also being used in the fixed width column type concatenation because I am now using `allocate_like` for column creation like you suggested earlier.

Reply:
Ah, okay, just wanted to make sure there is good reason (and there is).